### PR TITLE
RN_Techpreview_RHEL9_2_37_0

### PR DESCRIPTION
**Pull request by rasika-chivate**
Add RHEL9 migration as Technical Preview in the Release Notes, as this is confusing the users.

### CODE REVIEW

**Review by puneet0191 — APPROVED** [review]

File: docs/release-notes/2.37.1.md [hunk 1]
```diff
@@ -11,24 +11,27 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
...
+        ```
+    3. If you are running PMM via a virtual appliance (OVF or AMI), use SSH to shell into the PMM server and run this command:
+        ```sh
+        curl -fsSL https://raw.githubusercontent.com/percona/pmm/main/scripts/authfix.sh  | /bin/bash
+        ```
 
-For more details see, [blogpost](https://percona.com/blog/pmm-authentication-bypass-vulnerability-fixed-in-2-37-1/).
+    For more details see, [blogpost](https://percona.com/blog/pmm-authentication-bypass-vulnerability-fixed-in-2-37-1/).
 
+- An Enterprise Linux 9 (el9) version of PMM was released through oversight. This was intended to be a [technical preview](https://docs.percona.com/percona-monitoring-and-management/details/glossary.html#technical-preview) as we prepared for the GA in  2.38.0. There are several known issues with this el9-based version. Thus, we do not recommend running it in production until the official GA announcement.
```
Comment (BupycHuk) [inline docs/release-notes/2.37.1.md:35]: I think we shouldn't mention that it will be GAed in 2.38.0 in case we won't be able to fix those issues.

**Review by BupycHuk — APPROVED** [review]

File: docs/release-notes/2.37.1.md [hunk 2]
```diff
@@ -11,24 +11,27 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
...
+        ```sh
+        curl -fsSL https://raw.githubusercontent.com/percona/pmm/main/scripts/authfix.sh  | /bin/bash
+        ```
 
-For more details see, [blogpost](https://percona.com/blog/pmm-authentication-bypass-vulnerability-fixed-in-2-37-1/).
+    For more details see, [blogpost](https://percona.com/blog/pmm-authentication-bypass-vulnerability-fixed-in-2-37-1/).
 
+- An Enterprise Linux 9 (el9) version of PMM was released through oversight. This was intended to be a [technical preview](https://docs.percona.com/percona-monitoring-and-management/details/glossary.html#technical-preview). There are several known issues with this el9-based version. Thus, we do not recommend running it in production until the official GA announcement.
+
+    The images from the `percona` Docker repository have been removed. However, those who want to test them can locate them in the `perconalab` Docker repository.
```
Comment (rnovikovP) [inline docs/release-notes/2.37.1.md:37]:
```suggestion
    The images from the `percona` Docker repository have been removed. However, those who want to test them can locate them in the `[perconalab](https://hub.docker.com/layers/perconalab/pmm-server/2.37.1-el9/images/sha256-da890362a138f40cad9cc2c0ffb66bdda0d307d438848a87ba619397754f4400?context=explore)` Docker repository.
```

**Comment by rnovikovP** [comment]
Also, this needs to be added to 2.37.0 RN

**Review by rnovikovP — APPROVED** [review]

**Review by shoffman-percona — COMMENTED** [review]
Looks good to me!

**Pull request merged**